lib/workers/repository/process/write.ts

```diff
@@ -56,34 +55,27 @@ export async function writeUpdates(
       // Stop processing other branches because base branch has been changed
       return 'automerged';
     }
-    let deductPrRemainingCount = 0;
     if (res === ProcessBranchResult.PrCreated) {
-      deductPrRemainingCount = 1;
+      incLimitedValue(Limit.PullRequests);
     }
     // istanbul ignore if
```

Comment:
I think those `istanbul ignore if` should be removed, so this is more well tested

Reply:
To the another hand, the whole conditional chain will be deleted in upcoming PR